packages/locale/src/localizeAddress.ts

```diff
@@ -1,7 +1,10 @@
 import { Address, Country } from '@bigcommerce/checkout-sdk';
 import { find, isEmpty } from 'lodash';
 
-import { LocalizedGeography } from '../geography';
+interface LocalizedGeography {
```

Comment:
Should we get access ti this interface outside of this file? (somewhere in other package)

Reply:
I've checked `LocalizedGeography` uses only `localizeAddress` function